examples/linear_wake/analysis_equal.py

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/env python3
```

Comment:
This is the script we discussed that compares 2 openPMD files and checks whether they contain the same data to some precision, right? Indeed this is nice and useful. Let's keep it in `linear_wake` for the moment as you suggest, and if we start using it a lot in the future we'll put it somewhere else (and add a few input options like `--field-var-list`, `--species-list`, `particle-var-list`, `--tolerance`, stuff like that).

Reply:
Not really, I had something way more general in mind, independent of coordinate and unit systems. This was just something quick.

Reply:
Would a new folder be better than just a new `inputs_ion_motion_SI` file in `linear_wake`?

Reply:
You're right, new input file is probably better.